README.md

```diff
@@ -102,12 +101,14 @@ Rename the file to `baserom.z64`, `baserom.n64` or `baserom.v64`, depending on t
 
 #### 4. Make and Build the ROM
 
-To start the extraction/build process, run the following command:
+For a first time build, run the following command, where `N` is the number of cores your processor has (see the note at the bottom for more information):
 
 ```bash
-make init
+make init -j N
 ```
 
+After the very first build you'll want to run `make -j N` instead, since you likely won't need all the setup the `init` command provides.
+
```

Comment:
I wanted to mention about using `-jN` because most people will just run this command without reading the footnotes.

I also wanted to mention about not needing to run `make init` after the first run.